src/Console/Command/CleanBranchCommand.php

```diff
@@ -60,7 +62,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             'jiraPassword' => $config['CleanBranch']['password']
         ]));
 
-        $io->title('Scan folder ' . $folder . ' for outdated issues');
```

Comment:
you sure?

Reply:
Defo!
That's the CleanBranch Command which purges orphaned folders. ;)

Reply:
Oh wow.... them tomatoes! Now I see it 👍